### ISSUE TITLE
DK starter quest "the light of dawn" bug

### ISSUE THREAD

**Issue by Nussbaumerli** [FireFly-WoW/FireFly-IssueTracker#395]
**Description:**
The final dk quest "the light of dawn" does not end and the battle continues until darion mograine dies.
CHANGEME: Describe the issue here

**Current behaviour:**

CHANGEME: Explain to us (if/how) it currently works.

**Expected behaviour:**

CHANGEME: Explain to us how the issue is supposed to work.

**Steps to reproduce the problem:**

1. Step 1
2. Step 2
3. Step 3

**Screenshots:**

url...
url...
url...

**Comment by Siderbas** [comment]
duplicate of #338

**Comment by Shrizz** [comment]
I was able to complete this without any issue the other day. Although I remember speaking with someone who described this issue, it doesn't seem to always occur. I'm not sure if it affects anything, but during the battle I died numerous times and killed the NPC who is casting Tranquility constantly, several times (it just respawns instantly so doesn't really matter what you do). The battle ended when I got the enemy force counter down to around 200 and then the RP event began, but I don't remember if it's a timed event or if what you do actually matters in the end.
